rskj-core/src/main/java/org/ethereum/vm/trace/ProgramTraceProcessor.java

```diff
@@ -53,7 +64,11 @@ public JsonNode getProgramTracesAsJsonNode(List<Keccak256> txHashes) {
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList());
 
-        return OBJECT_MAPPER.valueToTree(txTraces);
+        SimpleFilterProvider filterProvider = new SimpleFilterProvider();
+        filterProvider.addFilter("opFilter",
+                SimpleBeanPropertyFilter.serializeAllExcept(traceOptions.getDisabledFields()));
+
+        return OBJECT_MAPPER.setFilterProvider(filterProvider).valueToTree(txTraces);
```

Comment:
multiple threads may collide here. `OBJECT_MAPPER` is `static` one, so setting a filter here may not be a thread-safe operation.

Reply:
I've changed that for THIS method only. 💪🏼

Reply:
(So the rest of them can perform as always)